guides/common/modules/proc_configuring-an-external-dhcp-server.adoc

```diff
@@ -166,7 +166,7 @@ Note that the IP address that you enter is the {Project} or {SmartProxy} IP addr
 # exportfs -rva
 ----
 
-. Configure the firewall for the DHCP omapi port 7911:
+. Configure the firewall for DHCP omapi port 7911:
 +
 [options="nowrap"]
 ----
```

Comment:
comment on line 173: it's probably a good idea to drop "\ &&" in favor of starting the second line with "# " so that users run two commands. That's AFAIK our general way.

Reply:
Addressed :+1: 
And also removed `double quotes` for adding ports in command. I guess, it was not necessary.